plugins/filter/cidr_merge.py

```diff
@@ -120,6 +120,7 @@ class mac_linux(netaddr.mac_unix):
 
 RETURN = """
   data:
+    type: raw
```

Comment:
As a safety measure, we can use validation.check_type_* methods to ensure that the input is only either str or list.
For example, https://github.com/ansible-collections/cisco.nxos/blob/main/plugins/action/nxos_file_copy.py#L88-L95.